src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -547,6 +547,11 @@ class ZaakZoekSerializer(serializers.Serializer):
         required=False,
         help_text=_("Array van zaaktypen."),
     )
+    zaaktype__not_in = serializers.ListField(
```

Comment:
@joeribekker @annashamray in https://github.com/open-zaak/open-zaak/issues/1783#issuecomment-2370776330 it's mentioned that this filter should be added to both `zaak_zoek` and `zaak_list`, but `zaaktype__in` is only available on `zaak_zoek` (https://github.com/VNG-Realisatie/gemma-zaken/issues/2072#issuecomment-1198244150), so shouldn't we keep `zaaktype__not_in` in line with this?

Reply:
also, should I mark it as experimental?

Reply:
Yes and yes